tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -154,6 +163,16 @@ pfcwd/test_pfc_config.py::TestPfcConfig::test_forward_action_cfg:
      reason: "Forward action not supported in cisco-8000"
      conditions:
         - "asic_type in ['cisco-8000']"
+
+#######################################
+#####        auto negotiation     #####
```

Comment:
It would be more consistent if the comment here can be `platform_tests`.

Reply:
sure